website/source/docs/upgrade-specific.html.md

```diff
@@ -96,6 +98,22 @@ with the the new ACL [Token](/api/acl/tokens.html) and
 
 More complete details on how to upgrade "legacy" tokens is available [here](/docs/guides/acl-migrate-tokens.html).
 
+### Connect Multi-datacenter
+
+This only applies to users upgrading from an older version of Consul Enterprise to Consul Enterprise 1.4.0 (all license types).
+
+In addition, this upgrade will only affect clusters where [Connect is enabled](/docs/connect/configuration.html) on your servers before the migration.
+
+Connect multi-datacenter uses the same primary/secondary approach as ACLs and will use the same [primary_datacenter](#primary-datacenter). When a secondary datacenter server restarts with 1.4.0 it will detect it is not the primary and begin an autotmatic bootstrap of multi-datacenter CA federation.
```

Comment:
typo: autotmatic -> automatic

Reply:
```suggestion
Connect multi-datacenter uses the same primary/secondary approach as ACLs and will use the same [primary_datacenter](#primary-datacenter). When a secondary datacenter server restarts with 1.4.0 it will detect it is not the primary and begin an automatic bootstrap of multi-datacenter CA federation.
```